src/JWT/Builder/JwtHeader.cs

```diff
@@ -52,5 +52,10 @@ public class JwtHeader
         [JsonPropertyName("x5t")]
 #endif
         public string X5t { get; set; }
+
+#if MODERN_DOTNET

```

Comment:
Please move it to the top of the class, so the ctor(s) would be before properties but after fields